ffi/capi/src/fixed_decimal.rs

```diff
@@ -40,7 +41,7 @@ pub mod ffi {
 
     /// Increment used in a rounding operation.
     #[diplomat::rust_link(fixed_decimal::RoundingIncrement, Enum)]
-    pub enum ICU4XRoundingIncrement {
+    pub enum ICU4XFixedDecimalRoundingIncrement {
```

Comment:
issue: this is a released stable API, we cannot rename it.

Reply:
Was thinking of undoing the rename and adding a comment about renaming it for 2.0, but maybe it would be better to `#[deprecate]` the old name instead. I think that would be harder to miss for the 2.0 release. What do you think?

Reply:
leave a comment. we're going to remove all the `ICU4X` prefixes in 2.0, so we'll touch this code.